src/components/cc-orga-member-list/cc-orga-member-list.types.d.ts

```diff
@@ -0,0 +1,52 @@
+import { OrgaMemberCardState, OrgaMemberRole, DeleteMember } from '../cc-orga-member-card/cc-orga-member-card.types';
+
+export interface OrgaMemberInviteFormState {
+  state: 'idle' | 'inviting';
+  email: EmailFormField;
+  role: RoleFormField;
+}
+
+interface EmailFormField {
+  value: string;
+  error?: 'empty' | 'invalid' | 'duplicate' | null;
+}
+
+interface RoleFormField {
+  value: OrgaMemberRole;
+}
+
+export type OrgaMemberListState = OrgaMemberListStateLoading | OrgaMemberListStateLoaded | OrgaMemberListStateLeaving | OrgaMemberListStateError;
+
+interface OrgaMemberListStateLoading {
+  state: 'loading';
+}
+
+interface OrgaMemberListStateLoaded {
+  state: 'loaded';
+  value: OrgaMemberCardState[];
+  identityFilter: string;
+  lastAdminLeavingError: boolean;
+  mfaDisabledOnlyFilter: boolean;
+}
+
+interface OrgaMemberListStateLeaving {
+  state: 'leaving';
+  value: OrgaMemberCardState[];
+  identityFilter: string;
+  mfaDisabledOnlyFilter: boolean;
+}
+
+interface OrgaMemberListStateError {
+  state: 'error';
+}
+
+interface InviteMember {
```

Comment:
I have seen that InviteMember is used in the event for both of the components and you import DeleteMember here.
InviteMember is not present in the card types though.

With the current plugin and for the future I would suggest that you put both InviteMember and DeleteMember in the common type file and we will import them in the component directly when needed.

Reply:
`DeleteMember` is used by both indeed so I'll move it to common :+1: 
`InviteMember` is only used by the `cc-orga-member-list` so I'll leave it as it is if that's ok with you :+1: